### DISCUSSION TITLE
Welcome to ttc Discussions!

### DISCUSSION THREAD

**Discussion by TudbuT** [TudbuT/ttc#1] [General]
## Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hecker12423** [comment]
How can I download TTC?

**Reply by WooHooASDF** [reply]
You can download ttc from [here](https://github.com/TudbuT/ttc/releases)

**Comment by luxuwu** [comment]
its an ip grabber btw

**Reply by TudbuT** [reply]
Its not
